openverse_api/catalog/api/serializers/audio_serializers.py

```diff
@@ -211,7 +211,9 @@ def create(self, validated_data):
 
 class AudioWaveformSerializer(serializers.Serializer):
     len = serializers.SerializerMethodField()
-    points = serializers.ListField(serializers.FloatField(min_value=0, max_value=1))
+    points = serializers.ListField(
```

Comment:
I believe #411 also has this change - just a heads up as that's likely to cause some merge conflicts

Reply:
Thanks for pointing that out, I'll remove this change if #411 gets merged before this one. It shows lots of warnings in Sphinx otherwise.